javascript/sdk/src/index.ts

```diff
@@ -42,6 +43,29 @@ export class ForeverVM {
     return await response.json()
   }
 
+  async *#getStream(path: string) {
+    const response = await fetch(`${this.#baseUrl}${path}`, { headers: this.#headers })
+    if (!response.ok) {
+      throw new Error(`HTTP error! status: ${response.status}`)
+    }
+
+    const decoder = new TextDecoder()
+    const reader = response.body!.getReader()
+    while (true) {
+      const { done, value } = await reader.read()
+      if (done) return
+
+      const lines = decoder
+        .decode(value)
+        .split('\n')
+        .filter((line) => line.trim())
+      for (const line of lines) {
+        console.log('LINE', line)
```

Comment:
was this intended to make it into the PR, or just for testing?

Reply:
whoops, just for testing